CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Add comment about being based on https://www.contributor-covenant.org/version/1/4/code-of-conduct.md

Reply:
Nevermind--it's at the bottom!